src/main/java/org/mitre/synthea/engine/Generator.java

```diff
@@ -919,4 +933,59 @@ private Predicate<String> getModulePredicate() {
   public RandomNumberGenerator getRandomizer() {
     return this.populationRandom;
   }
+
+  public void reduceNumberOfHospitals() {
+    // +1 Since this includes the header
+    int hospitalNumber = Integer.parseInt(Config.get("verily.limit_hospital_number")) + 1;
```

Comment:
why are we adding +1 to the limit? The limit is 4, but we set it to 5 here.
If we want the limit to be 5, why not just set that in the properties?

Reply:
Because by flag (synthea.properties) you'll want to limit the hospital number, to follow this example, to 4. Since you'll also want to load the header, that means you'll need a +1. 
If you take out the +1, you'll need to add it via input and I think this way is more clear on the ammount of hospitals you want to limit it to